src/__tests__/InputForm.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import InputForm from '../components/InputForm';
+
+describe('<InputForm />', () => {
+  const changeInput = () => {};
+  const fetchLocation = () => {};
+  const changeStatus = () => {};
+
+  it('disables submit button with no input text', () => {
+    const wrapper = shallow(
+      <InputForm
+        inputText={''}
+        changeInput={changeInput}
+        fetchLocation={fetchLocation}
+        changeStatus={changeStatus}
+      />,
+    );
+    const button = wrapper.find('input[type="submit"]');
+    expect(button.props().disabled).toEqual(true);
+  });
+
+  it('enables submit button with input text', () => {
+    const wrapper = shallow(
+      <InputForm
+        inputText={'Austin, TX'}
+        changeInput={changeInput}
+        fetchLocation={fetchLocation}
+        changeStatus={changeStatus}
+      />,
+    );
+    const button = wrapper.find('input[type="submit"]');
+    expect(button.props().disabled).toEqual(false);
```

Comment:
I tried `.not` but the `disabled` prop always exists, it just toggles between true and false.